arangod/Graph/TraverserDocumentCache.cpp

```diff
@@ -149,8 +149,8 @@ void TraverserDocumentCache::insertDocument(
           cache::CachedValue::construct(key, keySize, resVal, resValSize));
 
       if (value) {
-        bool success = _cache->insert(value.get());
-        if (!success) {
+        auto result = _cache->insert(value.get());
+        if (!result.ok()) {
           LOG_TOPIC(DEBUG, Logger::GRAPHS) << "Insert document into cache failed";
```

Comment:
I think @mchacki had it here in the initial commits with the cache. I'm unsure if there is a more appropriate topic? But it seems like it's probably fine as it's the `TraverserCache`.